specification.md

```diff
@@ -761,7 +761,7 @@ If the status report is processed successfully by the Server then the
 be populated as necessary.
 
 Here is the sequence diagram that shows how status reporting works (assuming
-server-side processing is successful):
+Server-side processing is successful):
```

Comment:
In this instance, I think it's weird to have capitalized "Server-side"

```suggestion
server-side processing is successful):
```

Reply:
I agree. Reverted.